drivers/usb_c/tcpc/fusb307.c

```diff
@@ -0,0 +1,770 @@
+/*
+ * Copyright (c) 2025 Jianxiong Gu <jianxiong.gu@outlook.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/usb_c/tcpci_priv.h>
+#include <zephyr/usb_c/usbc.h>
+#include <zephyr/usb_c/tcpci.h>
+#include <zephyr/shell/shell.h>
+#include "fusb307.h"
+
+#define DT_DRV_COMPAT onnn_fusb307_tcpc
+LOG_MODULE_REGISTER(tcpc_fusb307, CONFIG_USBC_LOG_LEVEL);
+
+/** Data structure for device instances */
```

Comment:
Comment style must be /* */, it is documented.
Also, there is no need to comment obvious things like `/* Data structure for device instances */`, `Delayable work`, `Boolean valu`, `State of CC1`...